src/components/badges/BadgeSection.tsx

```diff
@@ -15,7 +15,7 @@ const badgesList = classnames(
   display('grid'),
   gap('gap-2'),
   gridAutoRows('auto-rows-auto'),
-  gridTemplateColumns('grid-cols-1', 'lg:grid-cols-3')
+  gridTemplateColumns('grid-cols-1', 'md:grid-cols-3', 'lg:!grid-cols-6')
```

Comment:
So, basically we can have 1, 2 or 3 elements in a row: 2*3=6

Reply:
What's the reasoning for `md:grid-cols-3`? Wouldn't that cause the stretching to fail at medium screen sizes?

Reply:
Just tested it, I guess it doesn't :shrug: